GPU/GPUTracking/Global/GPUErrors.cxx

```diff
@@ -47,6 +47,17 @@ void GPUErrors::clear()
   memset(mErrors, 0, GPUCA_MAX_ERRORS * sizeof(*mErrors));
 }
 
+static std::unordered_map<uint32_t, const char*> errorNames = {
```

Comment:
Can you move this as static variable inside getErrorNames, so that it is not a global symbol?
Alternatively, please put it in o2::gpu::internal namespace.